vitess.io/css/site.scss

```diff
@@ -5,12 +5,12 @@
 
 // Colors
 // We're using the Material "Indigo" palette: https://material.google.com/style/color.html#color-color-palette
```

Comment:
I think you can remove this comment.

Can you please instead comment why you chose these colors and where you got them from?